src/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * This action type will be dispatched when your history
+ * receives a location change.
+ */
+export const LOCATION_CHANGE = '@@router/LOCATION_CHANGE'
```

Comment:
I'm not sure keeping the old name is worth the confusion generated by introducing two actions.

Especially now that we encourage calling history methods directly, people relying on the old action and thinking they don't need to change their analytics code will have broken code because direct history calls will not invoke the new action.

I think CALL_HISTORY_METHOD might be the best choice here. It makes it clear this is not intended for analytics.